website/src/components/MainLayout.jsx

```diff
@@ -7,10 +7,13 @@ import '../scss/main.scss';
 
 const muiTheme = getMuiTheme({
   palette: {
-    primary1Color: '#1f786a',
```

Comment:
Since this was for the muiTheme and not a style const, I went with the order listed on the Material UI site (by the primary colors, followed by accent, etc.)
I can change it though if you want!